tests/bpftrace.cpp

```diff
@@ -313,15 +295,34 @@ TEST(bpftrace, add_probes_wildcard_kprobe_multi)
 
 TEST(bpftrace, add_probes_wildcard_no_matches)
 {
-  ast::Probe *probe = parse_probe(
-      "kprobe:sys_read,kprobe:not_here_*,kprobe:sys_write{}");
+  auto bpftrace = get_strict_mock_bpftrace();
+  bpftrace->feature_ = std::make_unique<MockBPFfeature>(true);
+  EXPECT_CALL(*bpftrace->mock_probe_matcher,
+              get_symbols_from_traceable_funcs(false))
+      .Times(1);
+
+  parse_probe("kprobe:sys_read,kprobe:not_here_*,kprobe:sys_write{}",
+              *bpftrace);
 
+  ASSERT_EQ(2U, bpftrace->get_probes().size());
+  ASSERT_EQ(0U, bpftrace->get_special_probes().size());
+
+  std::string probe_orig_name =
+      "kprobe:sys_read,kprobe:not_here_*,kprobe:sys_write";
+  check_kprobe(bpftrace->get_probes().at(0), "sys_read", probe_orig_name);
+  check_kprobe(bpftrace->get_probes().at(1), "sys_write", probe_orig_name);
+}
+
+TEST(bpftrace, add_probes_wildcard_no_matches_kprobe_multi)
```

Comment:
(naming) Isn't the test above the one using kprobe_multi and this one is without it?

Reply:
Correct, good catch!